lib/routes/account.js

```diff
@@ -959,10 +1003,13 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+          return db.sessionTokenVerified(sessionToken.id)
+          .then( function(token) {
+            return {
+              email: token.email,
+              verified: !!(token.emailVerified && ! token.tokenVerificationId)
```

Comment:
Went with `sessionVerified` and `emailVerified`, seems to be a little more clear.